site/src/docs/components/date-input/customisation.mdx

```diff
@@ -0,0 +1,95 @@
+
+See all available CSS variables in the table below.
+
+| CSS variable               | Description                         |
+| ---------------------------- | ----------------------------------- |
+| `--date-background`          | Background colour for date          |
+| `--date-color`               | Font colour for date                |
+| `--date-border`              | Border for date                     |
+| `--selected-date-background` | Background colour for selected date |
+| `--selected-date-color`      | Font colour for selected date       |
+
+### Customisation example
+
+In this example some days have a different background colour to indicate that they have limited timeslots available. When days have colours to indicate something, it's important to include `legend` and `setDateAriaDescribedBy` props to inform the users of their meaning. Use the `setDateAriaDescribedBy` callback function to return corresponding LegendItem's element id for specific dates.
+
+<Playground scope={{ DateInput, useState, format }}>
+
+```jsx
+import { DateInput } from 'hds-react';
+
+() => {
+  const dateFormat = 'dd.M.yyyy';
+  const dateValue = new Date(2021, 10, 12);
+  const [value, setValue] = useState(format(dateValue, dateFormat));
+  const helperText = `Custom styles for days with limited available timeslots in the date picker calendar. Use format D.M.YYYY.`;
+
+  const littleSpaceLeftDate = {
+    elementId: 'little-space-left',
+    label: 'Only a few free timeslots available',
+    color: 'var(--color-summer-medium-light)',
+  };
+
+  const dayHasLittleSpace = (day) =>
+    day === 3 || day === 4 || day === 17 || day === 19 || day === 23 || day === 24;
+
+  const getDateLegendId = (date) => {
+    const day = date.getDate();
+    if (dayHasLittleSpace(day)) return littleSpaceLeftDate.elementId;
+    return undefined;
+  };
+
+  const setDateClassName = (date) => {
+    const legendId = getDateLegendId(date);
+    if (legendId === littleSpaceLeftDate.elementId) return 'little-space-left';
+    return undefined;
+  };
+
+  const setDateAriaDescribedBy = (date) => {
+    return getDateLegendId(date);
+  };
+
+  const legend = [littleSpaceLeftDate];
+  return (
+    <>
+      <style>
+        {`
+          .little-space-left {
+              --date-background: var(--color-summer-medium-light);
+              --date-border: 1px solid black;
+              --horizontal-spacing: 1px;
+          }
+        `}
+      </style>
+      <DateInput
+        value={value}
+        onChange={setValue}
+        helperText={helperText}
+        setDateClassName={setDateClassName}
+        setDateAriaDescribedBy={setDateAriaDescribedBy}
+        legend={legend}
+        style={{ width: 400 }}
+      />
```

Comment:
400px?

Reply:
I just set some value so the input isn't super wide.

Reply:
Sorry, I meant should the value be "400px" instead of "400" 😄

Reply:
I see! Set it to pixels.